CMakeLists.txt

```
@@ -4,8 +4,8 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
 # Project setup
 project(
-    kamping
-    DESCRIPTION "An MPI wrapper which makes using MPI feel like C++"
+    KaMPIng
+    DESCRIPTION "(Near) zero-overhead C++ MPI bindings."
```

```suggestion
    DESCRIPTION "Flexible and (near) zero-overhead C++ bindings for MPI"
```
Use same description as in the paper title (with normal capitalization).